Mapper UnMapped properties

### Describe

Mapper에서 변환해주는 Entity와 Aggreate에 매칭되지 않는 부분이 존재한다.

### How

_No response_

### Environment

_No response_

### Additional

```
/Users/leeseojune/Documents/Project/v1-service-notification/notification-infrastructure/build/tmp/kapt3/stubs/main/io/github/v1servicenotification/domain/setting/mapper/SettingMapper.java:10: warning: Unmapped target properties: "settingId, isActivated".
    public abstract io.github.v1servicenotification.domain.setting.domain.SettingEntity settingDomainToEntity(@org.jetbrains.annotations.NotNull
                                                                                        ^
/Users/leeseojune/Documents/Project/v1-service-notification/notification-infrastructure/build/tmp/kapt3/stubs/main/io/github/v1servicenotification/domain/setting/mapper/SettingMapper.java:14: warning: Unmapped target properties: "userId, notificationCategoryId, isActivated".
    public abstract io.github.v1servicenotification.setting.Setting settingEntityToDomain(@org.jetbrains.annotations.NotNull
                                                                    ^
/Users/leeseojune/Documents/Project/v1-service-notification/notification-infrastructure/build/tmp/kapt3/stubs/main/io/github/v1servicenotification/domain/detail/mapper/DetailMapper.java:10: warning: Unmapped target properties: "isRead, categoryEntity".
    public abstract io.github.v1servicenotification.domain.detail.domain.DetailEntity detailDomainToEntity(@org.jetbrains.annotations.NotNull
                                                                                      ^
/Users/leeseojune/Documents/Project/v1-service-notification/notification-infrastructure/build/tmp/kapt3/stubs/main/io/github/v1servicenotification/domain/detail/mapper/DetailMapper.java:14: warning: Unmapped target properties: "isRead, notificationCategoryId".
    public abstract io.github.v1servicenotification.detail.Detail detailEntityToDomain(@org.jetbrains.annotations.NotNull
```

--- a/notification-infrastructure/src/main/kotlin/io/github/v1servicenotification/domain/detail/mapper/DetailMapper.kt
+++ b/notification-infrastructure/src/main/kotlin/io/github/v1servicenotification/domain/detail/mapper/DetailMapper.kt
@@ -3,9 +3,16 @@ package io.github.v1servicenotification.domain.detail.mapper
 import io.github.v1servicenotification.detail.Detail
 import io.github.v1servicenotification.domain.detail.domain.DetailEntity
 import org.mapstruct.Mapper
+import org.mapstruct.Mapping
+import org.mapstruct.Mappings
 
 @Mapper
 interface DetailMapper {
     fun detailDomainToEntity(detail: Detail): DetailEntity
+
+    @Mappings(
+        Mapping(source = "categoryEntity.id", target = "notificationCategoryId"),
+        Mapping(source = "read", target = "isRead")
+    )
     fun detailEntityToDomain(detailEntity: DetailEntity): Detail
 }

--- a/notification-infrastructure/src/main/kotlin/io/github/v1servicenotification/domain/setting/mapper/SettingMapper.kt
+++ b/notification-infrastructure/src/main/kotlin/io/github/v1servicenotification/domain/setting/mapper/SettingMapper.kt
@@ -1,11 +1,23 @@
 package io.github.v1servicenotification.domain.setting.mapper
 
+import io.github.v1servicenotification.domain.category.domain.CategoryEntity
 import io.github.v1servicenotification.domain.setting.domain.SettingEntity
+import io.github.v1servicenotification.domain.setting.domain.SettingId
 import io.github.v1servicenotification.setting.Setting
 import org.mapstruct.Mapper
+import org.mapstruct.Mapping
+import org.mapstruct.Mappings
 
 @Mapper
 interface SettingMapper {
+
+
     fun settingDomainToEntity(setting: Setting): SettingEntity
+
+    @Mappings(
+        Mapping(source = "settingId.userId", target = "userId"),
+        Mapping(source = "settingId.categoryEntity.id", target = "notificationCategoryId"),
+        Mapping(source = "activated", target = "isActivated")
+    )
     fun settingEntityToDomain(settingEntity: SettingEntity): Setting
 }

